plugins/thunderbolt/fu-thunderbolt-image.c

```diff
@@ -491,24 +491,77 @@ get_host_locations (guint16 id)
 	}
 }
 
+/*
+ * Finds optional multi controller (MC) entry from controller DROM.
+ * Returns TRUE if the controller did not have MC entry or the
+ * controller and image MC entries match. In any other case FALSE is
+ * returned and error is set accordingly.
+ */
+static gboolean
+compare_device_mc (const FuThunderboltFwObject *fw,
+		   const FuThunderboltFwObject *image,
+		   GError **error)
+{
+	FuThunderboltFwLocation image_mc_loc = { .section = DROM_SECTION, .description = "Multi Controller" };
+	FuThunderboltFwLocation fw_mc_loc = { .section = DROM_SECTION, .description = "Multi Controller" };
```

Comment:
Matter of taste, but some would prefer `= image_mc_loc` instead of repeating the literal info.

Reply:
Also, I think most of the code uses `controller`, as `fw` is a bit confusing (it can be the FW image).

Reply:
(It makes sense to change the parameter name of `get_device_locations()` too as now it gets two different `FuThunderboltFwObject` parameters)